src/js/modules/I18nLink.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Link, withRouteData } from 'react-static'
+import { i18nalizeLinkTo } from 'js/constants/languages'
+
+const I18nLink = (props) => {
+  const {langCode, to, ...rest} = props;
+  //TO DO -- only pass Link props, not all remaining ...rest
+  return <Link to={i18nalizeLinkTo({to, langCode})} {...rest} />
+}
+
+export default withRouteData(I18nLink);
```

Comment:
Why do we need to wrap `I18nLink` with the Higher Order Component for RouteData?

Reply:
Because we need the langCode, which the withRouteData HOC passes in as props.